docs-source/content/samples/simple/azure-kubernetes-service/_index.md

```diff
@@ -5,1017 +5,45 @@ weight: 8
 
-* Operating System: GNU/Linux, macOS or [WSL for Windows 10](https://docs.microsoft.com/windows/wsl/install-win10).
-* [Git](https://git-scm.com/downloads), use `git --version` to test if `git` works.  This document was tested with version 2.17.1.
-* [Azure CLI](https://docs.microsoft.com/cli/azure), use `az --version` to test if `az` works.  This document was tested with version 2.9.1.
-* [kubectl](https://kubernetes-io-vnext-staging.netlify.com/docs/tasks/tools/install-kubectl/), use `kubectl version` to test if `kubectl` works.  This document was tested with version v1.16.3.
-* [helm](https://helm.sh/docs/intro/install/), version 3.1 and later, use `helm version` to check the `helm` version.  This document was tested with version v3.2.4.
+#### Azure Kubernetes Service cluster
 
-##### Create Service Principal for AKS
+Azure Kubernetes Service makes it simple to deploy a managed Kubernetes cluster in Azure. AKS reduces the complexity and operational overhead of managing Kubernetes by offloading much of that responsibility to Azure. As a hosted Kubernetes service, Azure handles critical tasks like health monitoring and maintenance for you. The Kubernetes masters are managed by Azure. You only manage and maintain the agent nodes. As a managed Kubernetes service, AKS is free - you only pay for the agent nodes within your clusters, not for the masters.
 
-An AKS cluster requires either an [Azure Active Directory (AD) service principal](https://docs.microsoft.com/azure/active-directory/develop/app-objects-and-service-principals) or a [managed identity](https://docs.microsoft.com/azure/aks/use-managed-identity) to interact with Azure resources.
+To learn more, please see the [What is Azure Kubernetes Service?](https://docs.microsoft.com/en-us/azure/aks/intro-kubernetes).
 
-We will use a service principal to create an AKS cluster. Follow the commands below to create a new service principal.
+#### Domain home source types
 
-Please run `az login` first. Do set the subscription you want to work with. You can get a list of your subscriptions by running `az account list`.
+This sample demonstrates running the WebLogic cluster on AKS using two domain home types. The instructions for each are self contained and independent. This section lists the domain home source types recommended for use with AKS, along with some benefits of each. For complete details on domain home source types, see [Choose a domain home source type]({{< relref "/userguide/managing-domains/choosing-a-model/" >}}).
```

Comment:
The 'choosing-a-model' relref didn't work in my version of hugo:

[tbarnes-1 docs-source]$ ./runpublic.sh
Building sites … ERROR 2021/02/24 17:13:24 [en] REF_NOT_FOUND: Ref "/userguide/managing-domains/choosing-a-model/" from page "samples/simple/azure-kubernetes-service/_index.md": page not found
WARN 2021/02/24 17:13:24 .File.UniqueID on zero object. Wrap it in if or with: {{ with .File }}{{ .UniqueID }}{{ end }}

Please append "/_index.md" to the reference.